decidim/decidim
# Fix accountability notifications proposal title

#### :tophat: What? Why?
After the proposal titles became translatable, the change was not reflected to the accountability notifications that refer to the proposals. The notifications include the whole translation hash for the proposal titles.

#### :pushpin: Related Issues
- Related to #6285

#### Testing
- Create a proposal and make sure you follow it
- Make sure you have email notifications enabled
- Create an accountability result and link it to the proposal
- Update the accountability result
- See the mailbox after the update

#### :clipboard: Checklist
